### PR TITLE
Seond pull request

### PR DESCRIPTION
**Pull request by Bluepoisson**
Corrected error forking from wrong repository by copying file/items to correct repo and new pull request.

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,209 @@
...
+    <div class="container">
+      <div class="post-wrap">    
+      <div class="post">
+       <div class="post-front">
+       <div class="post-img" style="background-image: url(./Cat-crops-5.jpg);" alt="Falling Cat"></div>
+       <div class="post-info">
+       <span>Active Speakers</span>
+       <h2>Genelecly Sound</h2>
+       </div>
+       </div>
```
Comment (bealun) [inline code/index.html:49]: Check the indentation so it will be easier to read and understand!

File: code/index.html [hunk 2]
```diff
@@ -1,22 +1,209 @@
...
+        </p>
+        </div>
+        <a href="#">Read More</a>
+        </div>
+        </div>
+        </div>
+      <div class="post-wrap">    
+        <div class="post">
+        <div class="post-front">
+        <div class="post-img" style="background-image: url(./sittingtiger.jpg);" alt="Woman in Tiger mask"></div>
```
Comment (bealun) [inline code/index.html:122]: Nice use of alt for the image 🙌

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,266 @@
-html {
-  background: #0025ff;
-  color: #fff;
+*   {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```
Comment (bealun) [inline code/style.css:4]: Nice so you don't have to repeat code 🌟

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,266 @@
...
+  height: 100%;
+  background-color: white;
+  backface-visibility: hidden;
+  border-radius: 10px;
+  box-shadow: 0 4px 6px -1px rgba(0,0,0,0.1);
+}
+
+.post-back   {
+  transform: rotateY(180deg);
+}
```
Comment (bealun) [inline code/style.css:113]: These are some really nice features, great job 👏

**Review by bealun — CHANGES_REQUESTED** [review]
You've done a great job this week! 👏 Sleek design, fantastic hover effects and a fun theme - really nice. I've left some comments for you to look at and the site requires some changes as it is not completely responsive. There's a lot of side scrolling that need a peek before you're approved and add some indentation - minor details. But it's still a really nice load of work you've put in. Be proud! 🎉

File: code/index.html [hunk 5]
```diff
@@ -1,22 +1,209 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>(H)ear Mag</title>
+    <link rel="icon" type="image/png" href="./Ear.png" sizes="16x16 32x32" >
+    <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+   <header>
+      <img class="logo" img src="./Ear.jpg" alt="Ear drawing">
```
[the remark this answers is not in the document]
Comment (Bluepoisson) [reply code/index.html]: Yes! I made one and for some reason deleted it.

File: code/index.html (hunk 1 above)
Comment (Bluepoisson) [reply code/index.html]: Yes! I was confused by the guidelines where the shown example had aligned html aligned.

File: code/index.html (hunk 2 above)
Comment (Bluepoisson) [reply code/index.html]: Thank you.

File: code/style.css (hunk 4 above)
Comment (Bluepoisson) [reply code/style.css]: Thank you.

**Comment by Bluepoisson** [comment]
> You've done a great job this week! 👏 Sleek design, fantastic hover effects and a fun theme - really nice. I've left some comments for you to look at and the site requires some changes as it is not completely responsive. There's a lot of side scrolling that need a peek before you're approved and add some indentation - minor details. But it's still a really nice load of work you've put in. Be proud! 🎉

Thank you very much. 
I struggled with the responsiveness.

**Review by bealun — APPROVED** [review]

**Pull request closed without merging**